scripts/UpdateDependencies.ps1

```diff
@@ -15,6 +15,12 @@ Import-Module -Scope Local -Force "$PSScriptRoot/common.psm1"
 Set-StrictMode -Version 1
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
+$GitHubEmail = $ConfigVars["GithubEmail"]
+$GitHubUsername = $ConfigVars["GithubUsername"]
+$GitHubPassword = $ConfigVars["GithubToken"]
+
+Set-GitHubInfo $GitHubPassword $GitHubUsername $GitHubEmail
```

Comment:
So, if I want to run this locally without creating a PR, how would I skip this? Might be useful to add a `-NoPush` and/or `-NoCommit` switch to skip this